pkg/columns/formatter/textcolumns/options.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package textcolumns
+
+type HeaderStyle int
+
+const (
+	HeaderStyleNormal HeaderStyle = iota
+	HeaderStyleUppercase
+	HeaderStyleLowercase
+)
+
+const (
+	DividerSpace = " "
+	DividerTab   = "\t"
+	DividerDash  = "—"
+)
+
+type Option func(*Options)
+
+type Options struct {
+	AutoScale      bool        // if enabled, the screen size will be used to scale the widths
+	ColumnDivider  string      // defines the string that should be used as spacer in between columns (default " ")
+	DefaultColumns []string    // defines which columns to show by default; will be set to all visible columns if nil
+	HeaderStyle    HeaderStyle // defines how column headers are decorated (e.g. uppercase/lowercase)
+	RowDivider     string      // defines the (to be repeated) string that should be used below the header
+	ShowRowDivider bool        // enables/disables the use of the row divider (below header)
+}
+
+func DefaultOptions() *Options {
+	return &Options{
+		AutoScale:      true,
+		ColumnDivider:  DividerSpace,
+		DefaultColumns: nil,
+		HeaderStyle:    HeaderStyleUppercase,
+		RowDivider:     DividerDash,
+		ShowRowDivider: true,
```

Comment:
I think the PR is ready to be merged, my main concern are the defaults values here:

- Do we want to start using a divider by default? 
- Do we want to autoscale by default?

I don't know, I think this PR should keep the same defaults for the UI and, in case, we can change those in the future actually to take advantage of all the features this library offers. 

cc @mauriciovasquezbernal @alban

Reply:
Agree, we should use what we have now and maybe change it later on. I set the defaults to AutoScale = false and RowDivider = DividerNone now. 👍

Reply:
I agree. Let's keep the same output we have now and then we can switch to something else.